mars/optimizes/runtime/optimizers/base_optimizer.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
hi, it can not rename to `core.py` , because `core.py` has **duplicate name** with [jax's file](https://github.com/google/jax/tree/110634d50dab0820aa777e161325f479f64db865/jax). It will cause wrong import as wisj said.

Reply:
OK, maybe just `base.py` ?

Reply:
OK, it's a better name.